akka-http-core/src/main/scala/akka/http/scaladsl/model/MediaType.scala

```diff
@@ -159,15 +269,24 @@ object MediaType {
     }
   }
 
-  def customMultipart(subType: String, params: Map[String, String]): Multipart = {
-    require(subType != "*", "Cannot create a MediaRange here, use MediaRanges.`multipart/*` instead!")
+  def customMultipart(
+    subType: String,
+    params:  Map[String, String]): Multipart = {
+    require(
+      subType != "*",
+      "Cannot create a MediaRange here, use MediaRanges.`multipart/*` instead!")
     new Multipart(subType, params)
   }
 
-  def custom(value: String, binary: Boolean, comp: Compressibility = Compressible,
-             fileExtensions: List[String] = Nil): MediaType = {
+  def custom(
+    value:          String,
+    binary:         Boolean,
+    comp:           Compressibility = Compressible,
+    fileExtensions: List[String]    = Nil): MediaType = {
     val parts = value.split('/')
-    require(parts.length == 2, s"`$value` is not a valid media-type. It must consist of two parts separated by '/'.")
+    require(
+      parts.length == 2,
+      s"`$value` is not a valid media-type. It must consist of two parts separated by '/'.")
     if (binary) customBinary(parts(0), parts(1), comp, fileExtensions)
     else customWithOpenCharset(parts(0), parts(1), fileExtensions)
   }
```

Comment:
Hmm ... not sure why this happened: something weird with scalafmt going on? I will try to revert the formatting.